Tribler/Core/Modules/restapi/multichain_endpoint.py

```diff
@@ -211,7 +211,8 @@ def render_GET(self, request):
         """
         Process the GET request which retrieves information about the multichain network.
 
-        .. http:get:: /multichain/network?focus_node=(string: public key)&neighbor_level=(int: neighbor_level)
+        .. http:get:: /multichain/network?dataset=(string: dataset)&focus_node=(string: public key)
```

Comment:
It's more of a descriptor of what this string has to represent rather than a variable. Hence I think that `public key` without the underscore is allowed.